### DISCUSSION TITLE
Can you please include point history

### DISCUSSION THREAD

**Discussion by Temian1** [cjmellor/level-up#47] [Ideas]
More like transaction to track how and when a point is added and removed
```
Can you add a function ( $user->transfer($user2 , 10)) the functionality to allow user to transfer or send point easily  to another user

**Comment by cjmellor** [comment]
Do you mean auditing? Because that is already a thing --  see: https://github.com/cjmellor/level-up#auditing

Regarding a method to transfer points... it's not something I personally am interested in spending any time on developing, but I am open to PR's for it. Maybe @mohamedalwhaidi would be interested.
